man/systemd.exec.xml

```diff
@@ -3163,6 +3163,40 @@ StandardInputData=V2XigLJyZSBubyBzdHJhbmdlcnMgdG8gbG92ZQpZb3Uga25vdyB0aGUgcnVsZX
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>Firewall Integration</title>
+    <variablelist class='unit-directives'>
+
+      <varlistentry>
+        <term><varname>DynamicUserNFTSet=</varname><replaceable>family</replaceable>:<replaceable>table</replaceable>:<replaceable>set</replaceable></term>
```

Comment:
this sounds awfully specific to me. I mean, I can see the usefulness, but why limit this to dynamic users? Why not do that for any UID here? it's really unnecessarily restricting to just say dynamic users.

Given this leaks into the unit file syntax, this really should be fixed before 252

Also, as in the other case I am sure the third part of the expression should be derived from the unit name automatically, if not specified, to push people to a "standardized" name of sets. And the examples provided should always make use of these default namings, instead of making up their own